src/main/java/org/openbase/bco/bcozy/BCozy.java

```diff
@@ -124,6 +125,15 @@ public void start(final Stage primaryStage) throws InitializationException, Inte
         BCozy.primaryStage = primaryStage;
         registerResponsiveHandler();
 
+        // TODO: should be removed after a synchronization error in the registry is fixed which
```

Comment:
Is this issue is already reported? If not please do so and refere the issue number to this todo.